app/services/dead_jobs/trs_register_for_trn.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module DeadJobs
+  class TrsRegisterForTrn < Base
+  private
+
+    # full class name to look for in Sidekiq dead jobs
```

Comment:
Sorry, little bugbear of mine, could we sentence-case all the comments please?
```suggestion
    # Full class name to look for in Sidekiq dead jobs
```